vm/src/stdlib/time.rs

```diff
@@ -325,6 +326,8 @@ mod time {
         tm_wday: PyObjectRef,
         tm_yday: PyObjectRef,
         tm_isdst: PyObjectRef,
+        tm_gmtoff: PyObjectRef,
```

Comment:
okay, now I see what you were talking about in the other pr, I missed this one, sorry! 

Yeah, these should be objects here, not options. The distinction between naive/aware is done based on their value (it being `None` or not), that's what I've understood by quickly looking into this. I'd need some more time to get a better view of this.